addon/components/modal.js

```diff
@@ -41,27 +33,11 @@ export default class ModalComponent extends Component.extend({
 	init() {
 		super.init(...arguments);
 
-		// Prevent creating an uncaught promise.
-		this.model.promise.catch(() => {}).finally(
-			this._close.bind(this),
-			`Component '${this.model.fullname}': close modal`
-		);
-	}
-
-	didInsertElement() {
-		super.didInsertElement(...arguments);
-
-		run.next(this.scheduler, 'scheduleOnce', this, '_open');
+		next(this, '_open');
 	}
 
-	didOpen() {}
-
 	_safeDidOpen() {
-		if (this.isDestroyed) {
```

Comment:
This cannot be removed. Restore the code or add the decorator `@isNotDestroyed` from https://github.com/BBVAEngineering/ember-modal-service/pull/153/files#diff-90faeac8c6232a7ed514e2b6d48ad58d4d9f453dcbdd116ab0648286ee997ea2R12

Same in other callbacks

Reply:
Done!